tests/results/SourceRangeTest.py

```diff
@@ -86,6 +86,18 @@ def test_json(self):
                                       end_line=4).__json__(use_relpath=True)
         self.assertEqual(uut['start'], self.result_fileB_line2)
 
+    def test_str(self):
+        uut = str(SourceRange.from_values('test_file', 1, 1, 2, 1))
+        compare = abspath('test_file') + ': FROM 1:1 TO 2:1'
+        self.assertEqual(uut, compare)
+
+        uut = str(SourceRange.from_values('test_file', 1, None, 2, None))
+        compare = abspath('test_file') + ': FROM 1 TO 2'
+        self.assertEqual(uut, compare)
+
+        uut = str(SourceRange.from_values('test_file', None, None, None, None))
+        self.assertEqual(uut, abspath('test_file'))
+
```

Comment:
Alright I will :smile: But, I was thinking one commit for each file would be neat?